packages/frontend/src/app/[locale]/campaigns/[chain]/[campaignId]/page.tsx

```diff
@@ -1,10 +1,11 @@
 import { CampaignDetails } from "@/src/components/campaign-details";
 import type { SupportedChain } from "@metrom-xyz/contracts";
+import type { Address } from "viem";
 
 export default function CampaignDetailsPage({
     params,
 }: {
-    params: { chain: SupportedChain; campaignId: string };
+    params: { chain: SupportedChain; campaignId: Address };
```

Comment:
This is not right. The campaign id is not an address but 32 bytes. We can use a `Hex` type from `viem`

Reply:
Fixed in 328249fe1c2965f43e5177cfc2abbaf5b371501d.